Core/WebServer.cpp

```diff
@@ -226,12 +255,30 @@ static void HandleListing(const http::ServerRequest &request) {
 	request.WriteHttpResponseHeader("1.0", 200, -1, "text/plain");
 	request.Out()->Printf("/\n");
 	if (serverFlags & (int)WebServerFlags::DISCS) {
-		// List the current discs in their recent order.
-		for (const std::string &filename : g_Config.RecentIsos()) {
-			std::string basename = RemotePathForRecent(filename);
-			if (!basename.empty()) {
-				request.Out()->Printf("%s\n", basename.c_str());
+		switch ((RemoteISOShareType)g_Config.iRemoteISOShareType) {
+		case RemoteISOShareType::RECENT:
+			// List the current discs in their recent order.
+			for (const std::string &filename : g_Config.RecentIsos()) {
+				std::string basename = RemotePathForRecent(filename);
+				if (!basename.empty()) {
+					request.Out()->Printf("%s\n", basename.c_str());
+				}
 			}
+			break;
+		case RemoteISOShareType::LOCAL_FOLDER:
+		{
+			std::vector<File::FileInfo> entries;
+			File::GetFilesInDir(Path(g_Config.sRemoteISOSharedDir), &entries);
+			for (const auto &entry : entries) {
+				// TODO: Support browsing into subdirs. How are folders marked?
```

Comment:
Folders have a trailing slash.  But I can't remember if browsing supports them... I kinda think it does?  It wouldn't be hard to anyway.

People will probably expect this to be recursive or else to support folders (my personal preference would be folders to work.)

-[Unknown]

Reply:
Yeah, I will add folder support later. I didn't think about the trailing slash - that'll be enough, and I'll make sure it works if it doesn't already.

Reply:
Hm, let's hope the user doesn't have a subfolder called "debugger". Since /debugger is handled separately :)